src/dmd/cli.d

```diff
@@ -314,7 +314,37 @@ dmd -cov -unittest myprog.d
             "same as -i=-std,-core,-etc,-object"
         ),
         Option("i=[-]<pattern>,[-]<pattern>,...",
-            "include/exclude imported modules whose name matches one of <pattern>"
+            "include/exclude imported modules whose name matches one of <pattern>",
+            q"{$(P Enables "include imports" mode, where the compiler will include imported
+	     modules in the compilation, as if they were given on the command line.)
+
```

Comment:
another subtle difference, again not clear if feature or bug but should be documented:

```
dmd -od=generated -op -c -Ifoo foo/fun1.d foo/bar/fun4.d foo/main.d
dmd -od=generated_i -op -c -i -Ifoo foo/main.d
```

```
foo/main.d: void main(){import bar.fun4; import fun1;}
foo/fun1.d:
foo/bar/fun4.d:module bar.fun4;
```

```
├── generated
│   └── foo
│       ├── bar
│       │   └── fun4.o
│       ├── fun1.o
│       └── main.o
├── generated_i
│   ├── bar
│   │   └── fun4.o
│   ├── foo
│   │   └── main.o
│   └── fun1.o

```

Reply:
yeah looks like a bug, I"ll see if I can fix this one